tests/test_clipboard.py

```diff
@@ -16,55 +16,42 @@ def test_unicode_clipboard():
 example_html = textwrap.dedent(
     """
     Version:0.9
-    StartHTML:71
-    EndHTML:170
-    StartFragment:140
-    EndFragment:160
-    StartSelection:140
-    EndSelection:160
-    <!DOCTYPE>
-    <HTML>
-    <HEAD>
-    <TITLE> The HTML Clipboard</TITLE>
-    <BASE HREF="http://sample/specs">
-    </HEAD>
-    <BODY>
-    <UL>
-    <!--StartFragment -->
-    <LI> The Fragment </LI>
-    <!--EndFragment -->
-    </UL>
-    </BODY>
-    </HTML>
+    StartHTML:00000138
+    EndHTML:00000215
+    StartFragment:00000171
+    EndFragment:00000182
+    StartSelection:00000174
+    EndSelection:00000178
+    <html><body>
+    <!--StartFragment--><p>😀</p><!--EndFragment-->
+    </body></html>
```

Comment:
I'm thinking here we should keep the original fixture and test and create a _new_ test that specifically uses multibyte characters, separate from this basic test. That will help assure that we haven't accidentally regressed (all the original checks are still in place), but also, the test can be named to indicate what it's protecting (something like `test_html_paste_with_multibyte_characters`).